torch/csrc/jit/codegen/cuda/python_frontend/fusion_record.h

```diff
@@ -420,13 +420,31 @@ struct PermuteOpRecord : RecordFunctor {
   void operator()(FusionDefinition& fd) final {
     auto arg =
         fd.getFusionState(args_.at(0).index)->template as<Nvf::TensorView>();
-    auto output = torch::jit::fuser::cuda::permute(arg, permutation_);
+    auto output = Nvf::permute(arg, order_);
     fd.setFusionState(outputs_.at(0).index, output);
   }
 
+  virtual void print(std::ostream& os, bool close_function = true) const {
```

Comment:
Should `RecordFunctor::print` be a pure virtual function? The current code could be moved to a helper function to be used in derived prints. So that we don't overlook omitted print next time.

Reply:
This is reasonable.  Do you mind if this cleanup is a separate PR?  Perhaps re-implementing the prints as a visitor pattern might make more sense.  I was just lazy in doing so, originally.

Reply:
Of course, there's no need to do it in this PR.